src/Core/Config/ConfigCacheLoader.php

```diff
@@ -193,10 +189,6 @@ private function loadINIConfigFile($filepath)
 	 */
 	private function loadConfigFile($filepath)
 	{
-		if (!file_exists($filepath)) {
```

Comment:
This one isn't necessary because:
- it's a private method
- it is called after a `file_exists()` check